app/js/controllers/service-tags.js

```diff
@@ -0,0 +1,14 @@
+import React from 'components/react';
+import ReactDOM from 'components/react-dom';
+import { Controller } from 'components/mvc';
+
+import ServiceTagsView from 'js/views/service-tags';
+
+export default class ServiceTagsController extends Controller {
+  main(id) {
+    ReactDOM.render(React.createElement(ServiceTagsView, {
+      id,
+      foxbox: this.foxbox,
+    }), this.mountNode);
+  }
+}
```

Comment:
Many controllers look the same with only the React tag changing.
I wonder if we can do something clever and avoid code repetition. Maybe using a kind of factory.
Anyway, that's not needed for this PR.